site/public/js/forum.js

```diff
@@ -1260,6 +1281,14 @@ function modifyThreadList(currentThreadId, currentCategoriesId, course, loadFirs
     });
 }
 
+// eslint-disable-next-line no-unused-vars
+function displayHistoryAttachment(edit_id) {
```

Comment:
Unused file eslint ignores should be listed at the top of the file as per the [documentation](https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript#style-rules).